src/main/java/mate/academy/MergeSortAction.java

```diff
@@ -1,8 +1,60 @@
 package mate.academy;
 
-public class MergeSortAction {
+import java.util.concurrent.RecursiveAction;
+
+public class MergeSortAction extends RecursiveAction {
+    private static final int THRESHOLD = 1;
+    private int start;
+    private int end;
+    private int[] array;
 
     public MergeSortAction(int[] array) {
+        this(0, array.length, array);
+    }
```

Comment:
When dealing with merge sort, the most common practice is to pass the subarray as a recursive param.
`start` and `end` shouldn't be necessary here - the less state you keep within the object, the better.